taurus/tests/test_examples.py

```diff
@@ -162,4 +162,4 @@ def test_access_data():
 
     # check that the serialization results in the correct number of objects in the preface
     # (note that neither measurements nor ingredients are serialized)
-    assert(len(json.loads(dumps(island))[0]) == 23)
+    assert(len(json.loads(dumps(island))[0]) == 26)
```

Comment:
Brendan has rightly pointed out that element count tests are weak.

Reply:
True, but I think that's out of scope.